### PR TITLE
Allow vpc peering when accepter doesn't have 1:1 subnet to routing table mapping

### PR DESCRIPTION
**Pull request by lofidellity**

### CODE REVIEW

**Comment by lofidellity** [comment]
Sorry for the commit spam in there, this includes changes from the previous PR as well.

File: accepter.tf [hunk 1]
```diff
@@ -85,14 +86,14 @@ locals {
...
-  subnet_id = "${element(local.accepter_subnet_ids, count.index)}"
+data "aws_route_tables" "accepter" {
+  count    = "${local.count}"
+  provider = "aws.accepter"
+  vpc_id   = "${local.accepter_vpc_id}"
 }
 
 locals {
-  accepter_aws_route_table_ids           = "${distinct(sort(data.aws_route_table.accepter.*.route_table_id))}"
+  accepter_aws_route_table_ids           = "${distinct(sort(data.aws_route_tables.accepter.ids))}"
```
Comment (aknysh) [inline accepter.tf]: @lofidellity  thanks for the PR!
did you test it (`terraform plan/apply`)?
does not look like `data "aws_route_tables"` supports `ids` output https://www.terraform.io/docs/providers/aws/d/route_table.html#attributes-reference

Usually Terraform does not have `ids`, but `id` when supported.

Comment (lofidellity) [reply accepter.tf]: https://www.terraform.io/docs/providers/aws/d/route_tables.html#attributes-reference is the correct document to look at. Yes this has been tested.

Comment (lofidellity) [reply accepter.tf]: Solution comes from this bug: https://github.com/terraform-providers/terraform-provider-aws/issues/715 @aknysh

Comment (aknysh) [reply accepter.tf]: oh sorry, did not notice `aws_route_tables` with an `s`

**Pull request merged**